openqa-maintenance.py

```diff
@@ -722,7 +559,11 @@ def trigger_build_for_target(self, prj, u):
 
         buildnr = "%s-%d" % (today, buildnr + 1)
 
-        for s in u['settings']:
+        for s in data['settings']:
+            if 'incidents' in data.keys():
+                incidents = self.calculate_incidents(data['incidents'])
```

Comment:
if calculate_incidents is a static method you don't need self. But I suggest not to make it static so you have the apiurl handy